pkg/client/e2e.go

```diff
@@ -46,7 +45,7 @@ func (*SonobuoyClient) GetTests(reader io.Reader, show string) ([]reporters.JUni
 			}
 			// TODO(chuckha) consider reusing this function for any generic e2e-esque plugin results.
 			// TODO(chuckha) consider using path.Join()
-			if path == e2eJunitPath || path == legacye2eJunitPath {
+			if path == e2eJUnitPath || path == legacye2eJUnitPath {
```

Comment:
Updated a bunch of little lines for consistent naming: `JUnit` preferred over `Junit`